DOC-2284: Add new documentation for `Revision History` Plugin. (#3071)
tinymce/tinymce-docs: modules/ROOT/partials/configuration/revisionhistory_css_url.adoc

[[revisionhistory_css_url]]
== `revisionhistory_css_url`

This option sets the location where a CSS file containing the styles for change annotations should be loaded from. It can be used along with `+revisionhistory_diff_classes+` to override the default CSS classes.

[NOTE]

> Ensure that the file includes all CSS classes expected by xref:revisionhistory_diff_classes[`+revisionhistory_diff_classes+`]

Reviewer: ```suggestion
Ensure that the file includes all CSS classes expected by xref:revisionhistory_diff_classes[`+revisionhistory_diff_classes+`].
```

> Make sure that the file contains for all the CSS classes which are expected by xref:revisionhistory_diff_classes.adoc[`+revisionhistory_diff_classes+`]

Reviewer 2: ```suggestion
Ensure that the file includes all CSS classes expected by xref:revisionhistory_diff_classes.adoc[`+revisionhistory_diff_classes+`]
```

[[revisionhistory_css_url]]
== `revisionhistory_css_url`

This option provides the default location to load the change annotations css styling from. This option can be used along with `+revisionhistory_diff_classes+` to provide custom diff classes from a css file.

////

> What does the option do?

Author: need more info here @tiny-ben-tran

Why use it?
When use it?
What values can it use?
What do these values do?
Are there risks?
  - Explain without using ‘risk’ or similar words.
  - Use NOTE or IMPORTANT admonitions if helpful.
  - For longer or more complicated scenarios, use the limitations section below.
////

*Type:* `+String+`

*Default value:* `+'${pluginUrl}/css/revisionhistory.css'+`

=== Example: using `revisionhistory_css_url`

[source,js]
----
tinymce.init({
  selector: 'textarea',  // change this value according to your HTML
  plugins: 'revisionhistory',
  toolbar: 'revisionhistory',
  revisionhistory_css_url: './revisionhistory.css'
});
----

// Remove if not applicable.
// Edit the sub-head to singular or plural as required.

> === Limitation(s) of the `revisionhistory_css_url` option

Author: any limitations for `revisionhistory_css_url`? @tiny-ben-tran
